clients/AlphabotClientLibrary/AlphabotClientLibrary/AlphabotClientLibrary.Shared/Models/BleConnectionData.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.NetworkInformation;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Remove unnecessary using directives.
```suggestion
using System.Net.NetworkInformation;
```